docs/UserGuide.md

```diff
@@ -24,24 +24,24 @@ Interested to use SoConnect? Then you are in the right place! In this user guide
 
 ## 3. Introduction
 
-SoConnect is a **2-in-1 desktop app for managing contacts and tasks**. It is originally built to help <a href="#nus">NUS</a> SoC students stay better connected to their school life, in terms of social connections and school tasks.
+SoConnect is a **2-in-1 desktop app for managing contacts and tasks**. It is originally built to help [NUS](#nus) SoC students stay better connected to their school life, in terms of social connections and school tasks.
 
-SoConnect leverages on the simplicity of Command Line Interface (<a href="#cli">CLI</a>) to optimize your productivity, while still maintaining user-friendliness through its Graphical User Interface (<a href="#gui">GUI</a>). If you can type fast, SoConnect can get your contact and task management work done faster than even the popular applications such as Microsoft People and To Do!
+SoConnect leverages on the simplicity of Command Line Interface ([CLI](#cli)) to optimize your productivity, while still maintaining user-friendliness through its Graphical User Interface ([GUI](#gui)). If you can type fast, SoConnect can get your contact and task management work done faster than even the popular applications such as Microsoft People and To Do!
```

Comment:
The `[GUI](#gui)` is working is it? Because I tried to link to the GUI but the intellij didn't give me any recommendation, I have to use `<a href="#gui">` instead.

Reply:
Yep, intellij doesn't recommend but if you manually type the correct id, it should work according to a previous test conducted